### ISSUE TITLE
Allow created load balancers to attach a floating IP

### ISSUE THREAD

**Issue by mildred** [digitalocean/digitalocean-cloud-controller-manager#272]
It would be great if load balancers created by the cloud-controller could attach themselves to specific floating IP addresses. It would allow pre-provisionning an IP address and a DNS record, and have the kubernetes cluster make itself available through it without manual intervention afterwards.

**Comment by timoreimann** [comment]
Hi @mildred 

thanks a lot for filing an issue.

Having floating IPs work on DO load-balancers would indeed be a nifty feature. Unfortunately, that's something that the LB product [does not support as of now](https://www.digitalocean.com/docs/networking/load-balancers/overview/#limits).

I'm happy to forward the feature request to the responsible team internally. The integration effort into CCM would be a follow-up step then.

Thanks again!

**Comment by h0jeZvgoxFepBQ2C** [comment]
Yeah this is really a big thing, it would be great if DO could support this!

**Comment by h0jeZvgoxFepBQ2C** [comment]
Any updates on this?
